extensions/interactions/DragAndDropSortInput/DragAndDropSortInput.py

```diff
@@ -38,7 +38,7 @@ class DragAndDropSortInput(base.BaseInteraction):
             'type': 'list',
             'validators': [{
                 'id': 'has_length_at_least',
-                'min_value': 2
+                'min_value': 1
```

Comment:
This backend schema decides how many editors will appear initially. This min_value is passed to the SchemaBasedEditor and then editors are rendered accordingly. Now, the frontend validation is taking place which disallows the number of choices that are less than 2.

Reply:
`SchemaBasedEditorDirective` gets the customisation schema for each interaction. Accordingly, the number of editors will appear as per the min_value. We can't set min_value- 2 and default value- 1 as customisation schema tests will fail in this case. Moreover, this schema editor is too generic and we can't make any change as it may affect at all other places as well. So, I could think of one effective solution of validating it in the frontend side but there's no such way to add this restriction in backend as well.